axelrod/_strategy_utils.py

```diff
@@ -45,11 +45,19 @@ def limited_simulate_play(player_1, player_2, h1):
     player_2's strategy method to set any internal variables as needed. If you
     need a more complete simulation, see `simulate_play` in player.py. This
     function is specifically designed for the needs of MindReader."""
-    h2 = player_2.strategy(player_1)
+    h2 = cheating_bastard(player_1, player_2)
     update_history(player_1, h1)
     update_history(player_2, h2)
 
 
+def cheating_bastard(cheater, opponent):
```

Comment:
Eeeesh 

> names:
> 
> inspect_strategy(inspector, opponent):
> foil_strategy_inspections(self, opponent):
> how do those sound?

That sounds good to me :+1:

> i also had issues understanding what the problem was. i was not aware of the variable declaration as an inline comment.

Eeeesh that sounds difficult, I can't offer any assistance I'm afraid...